src/mantine-dates/src/utils/get-end-of-week/get-end-of-week.test.ts

```diff
@@ -2,7 +2,10 @@ import { getEndOfWeek } from './get-end-of-week';
 
 describe('@mantine/dates/get-end-of-week', () => {
   it('returns end of the week', () => {
-    const endOfWeek = getEndOfWeek(new Date(2021, 1, 1));
+    let endOfWeek = getEndOfWeek(new Date(2021, 1, 1));
```

Comment:
It will be better to split this test into two tests: first test checks default behavior, second checks sunday as first day of week